pom.xml

```diff
@@ -783,12 +783,38 @@
                         <include>**/common/utils/RetryerUtilsTest.java</include>
                         <include>**/common/plugin/FilePluginManagerTest</include>
                         <include>**/common/plugin/PluginClassLoaderTest</include>
+                        <include>**/dao/AlertDaoTest.java</include>
+                        <include>**/dao/datasource/OracleDataSourceTest.java</include>
+                        <include>**/dao/datasource/HiveDataSourceTest.java</include>
+                        <!--<include>**/dao/entity/ProcessDefinitionTest.java</include>-->
+                        <include>**/dao/entity/TaskInstanceTest.java</include>
+                        <include>**/dao/entity/UdfFuncTest.java</include>
                         <include>**/dao/mapper/AccessTokenMapperTest.java</include>
                         <include>**/dao/mapper/AlertGroupMapperTest.java</include>
                         <include>**/dao/mapper/CommandMapperTest.java</include>
                         <include>**/dao/mapper/ConnectionFactoryTest.java</include>
                         <include>**/dao/mapper/DataSourceMapperTest.java</include>
-                        <include>**/dao/entity/UdfFuncTest.java</include>
+                        <include>**/dao/mapper/DataSourceUserMapperTest.java</include>
```

Comment:
Please revert the changes and just add your test.

Reply:
done. and you may wannt to know that there're two parts for `/dao` here and the `dao/entity/ProcessDefinitionTest` is missing.